src/services/meizam-api/meizam-api.ts

```diff
@@ -1,4 +1,6 @@
 import axiosWrapper  from './axios-wrapper'
+// import { UserPrediction } from '../../store/user/types';
```

Comment:
remove comment
as a rule of thumb, comments should be removed. for this reasons:
1) if its code that you "don't want to forget"  - we have the source control for that. so code never disappears.
2) if you need to explain the code, it probably means that it is too complicated and you clean it to be more readable without the need for comments or refactor to more classes, each is smaller. you can read about all these techniques in the super-recommended book "clean code".